api/src/main/java/com/cloud/network/NetworkModel.java

```diff
@@ -125,6 +125,10 @@ public interface NetworkModel {
      */
     String getNextAvailableMacAddressInNetwork(long networkConfigurationId) throws InsufficientAddressCapacityException;
 
+    default boolean isMACUnique(String mac, long networkId) {
+        return true;
+    }
```

Comment:
why this default implementation?

Reply:
@DaanHoogland there are some mock classes implementing it, not implemented it there (I think, we can return default true for them)

Reply:
yes, but it seems a logical error for anyone wanting to implement a new `NetworkModel`. They should be forced to implement it and not use this code by default.